frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -72,3 +72,10 @@ export const calcPVsCapacity = (pvs: K8sResourceKind[]): number =>
 
 export const getSCAvailablePVs = (pvsData: K8sResourceKind[], sc: string): K8sResourceKind[] =>
   pvsData.filter((pv) => getPVStorageClass(pv) === sc && pv.status.phase === status.AVAILABLE);
+
+export const getMinSizePVCapacity = (pvs: K8sResourceKind[]): number => {
```

Comment:
We need min size for the PV's that are not bound.

Reply:
Yup, pvs with available state only will come in the list

Reply:
ack